eng/pipelines/pipeline-generation.yml

```diff
@@ -93,11 +93,13 @@ jobs:
         Branch: master
         Prefix: cpp
         PublicOptions: ''
+        # Resource Provisioner is in InternalOptions and not TestOptions becasue C++ includes live tests in the unified pipelines
         InternalOptions: >-
           --variablegroup ${{ parameters.Release_Secrets_for_GitHub }}
           --variablegroup ${{ parameters.Release_Secrets_for_GitHubIO_Publishing }}
           --variablegroup ${{ parameters.Secrets_for_DevOps_Notification_Configuration_and_User_Resolution }}
           --variablegroup ${{ parameters.APIReview_AutoCreate_Configurations }}
+          --variablegroup ${{ parameters.Secrets_for_Resource_Provisioner }}
```

Comment:
We probably also need it at https://github.com/Azure/azure-sdk-tools/blob/6a036fa3628907185d0451a503d8cd2db33fb51a/eng/pipelines/pipeline-generation.yml#L96.

Reply:
I added a change that included that shortly before you commented I think. We don't have `- tests` pipelines given the C++ architecture.